tests/native/test_integration_native.py

```diff
@@ -260,6 +260,24 @@ def test_basic_arm(self):
         with open(os.path.join(workspace, "test_00000001.stdout")) as f:
             self.assertIn("Message", f.read())
 
+    def test_fclose_linux_amd64(self):
+        """
+        Tests that the fclose example for amd64 linux doesn't crash; see #1602 and #1604.
+        """
+        filename = os.path.abspath(os.path.join(DIRPATH, "binaries", "fclose_linux_amd64"))
+        workspace = os.path.join(self.test_dir, "workspace")
+        cmd = [
+            PYTHON_BIN,
+            "-m",
+            "manticore",
+            "--no-color",
+            "--workspace",
+            workspace,
+            filename,
+            "+++++++",
+        ]
+        subprocess.check_call(cmd, stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
```

Comment:
I think we should omit the `stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL` arguments because `pytest` will automatically hide the output unless there's a failure. However, even in the case of a failure, with the `DEVNULL` redirections, we won't get _any_ output.

Reply:
Good catch!

Reply:
P.S. the dev guide for Manticore suggests running `nosetests`, which does _not_ silence the output.

Reply:
That dev guide needs to be changed, it seems. We run `pytest` in CI, which means we should also be telling people to run `pytest` locally. Also, the last release for `nose` was in 2015, which is another good reason to use `pytest`.

Reply:
Let me see about updating that wiki page.